lib/internal/Magento/Framework/Mview/View/Subscription.php

```diff
@@ -209,7 +218,9 @@ protected function buildStatement($event, $changelog)
                     $describe = $this->connection->describeTable($tableName)
                 ) {
                     $columnNames = array_column($describe, 'COLUMN_NAME');
-                    $columnNames = array_diff($columnNames, $this->ignoredUpdateColumns);
+                    $ignoredColumns = array_merge($this->ignoredUpdateColumns,
```

Comment:
Formatting incorrect. If multi-line then all parameters must be on a line of their own - this is causing tests to fail.

Reply:
Have fixed in latest commit